local/compose/start.go

```diff
@@ -41,8 +49,31 @@ func (s *composeService) Start(ctx context.Context, project *types.Project, cons
 	if err != nil {
 		return err
 	}
-	if group != nil {
-		return group.Wait()
+
+	if options.Attach == nil {
+		return nil
+	}
+
+	eg, ctx := errgroup.WithContext(ctx)
+	for _, c := range containers {
+		c := c
+		eg.Go(func() error {
+			statusC, errC := s.apiClient.ContainerWait(ctx, c.ID, container.WaitConditionNotRunning)
+			select {
+			case status := <-statusC:
+				service := c.Labels[serviceLabel]
+				options.Attach.Status(service, getContainerNameWithoutProject(c), fmt.Sprintf("exited with code %d", status.StatusCode))
+				if options.Listener != nil {
+					options.Listener <- compose.Event{
+						Service: service,
+						Status:  int(status.StatusCode),
+					}
+				}
+				return nil
```

Comment:
Feels strange to just ignore exit code and return nil here. (In relation to not returning errors at https://github.com/docker/compose-cli/pull/1259/files#diff-057babf47f6c9d3e25ac4ca32267631c34ac2e0b816227d7278ef89418ac4818R95, not a good feeeling about this, intuitively)

Reply:
there's no error to return here. `ContainerWait`'s `errChan` is checked for API errors alrady, and here we just capture container exit Code
Same on https://github.com/docker/compose-cli/pull/1259/files#diff-057babf47f6c9d3e25ac4ca32267631c34ac2e0b816227d7278ef89418ac4818R95: attach just starts goroutines, then return